Usage/Run.md

```diff
@@ -0,0 +1,113 @@
+# Run
+
+- Java 8 or later is recommended for execution. (Not tested with lower versions.)
+- Run on the command line with the appropriate arguments.  
+   - `./CCFinderSW D -d directory -l language -o outputtxt`
+- The clone pair information output by CCFSW can be read by Gemini or CCFX.
+
+## [D] CloneDetector command line arguments  
+- -help
+   - A reasonable amount of help will be given.
+
+### Required Arguments
+- -d <arg> 
+   - Specify the directory path where the source files are located.  
+- -l <arg> 
+   - Specify the language name for code clone detection.  
+   - This language name is used when loading the options file.
+- -o <arg>
+   - Specify the output file name which contains clone pair information.  
+     - e.g., Entering `-o outputfile` will create *outputfile.txt*.
+   - If you are in ccfx mode, *outputfile.ccfxd* will be created.
+   - If you are in ccfsw mode, *outputfile_ccfsw.txt* is also created.
+
+### Optional Arguments
+#### lexical analysis and detection
+- -t \<arg>
+   - Specify the minimum number of tokens (threshold) of code clones to be detected.  
+   Specify a number, e.g. `-t 90`.
+   - The default value is 50. If no argument is given, the default value is used.
+- -w \<arg>
+   - Sets the range of clone pairs (0 <= detectionRange <= 2)
+   - 0 All range
+   - 1 within files only
+   - 2 Between files only
+- -b
+   - Recognizes code blocks with blank indentation.
+   - Cannot be used in antlr mode.
+- -antlr \<arg>
+   - Extracts information used for comment removal and identifier identification from the grammar files in `src/main/dist/grammarsv4`.
+   - Normal rules cannot be used.
+   - The argument must be a regular expression of the extension name to recognize the target source code.  (just put the extensions in a row with a vertical bar between).
+       - Example: -antlr h|hh|hpp|hxx|c|cc|cpp|cxx  
+                  -antlr py
```

Comment:
please rewrite l42-l43
```
       - Example1: -antlr h|hh|hpp|hxx|c|cc|cpp|cxx  
       - Example2: -antlr py
```

Reply:
Thanks for your review.
I have updated this part in the Japanese and English Readme files.